### PR TITLE
Add build script for windows, rename log4j file to not conflict

### PR DESCRIPTION
**Pull request by hdeadman**
CAS and CAS Management can't both have log4j2.xml in /etc/cas/config and the copy tasks in the build scripts will override the files for the other app (if they are both being tested on same server). 

log4j2-management xml also updated to use additivity for loggers and allow system properties to be used to specify log location and level.

### CODE REVIEW

File: etc/cas/config/management.properties [hunk 1]
```diff
@@ -1,12 +1,16 @@
+# CAS server that management app will authenticate with
+# The jasigcas server will authenticate for any app (service) and you can login as casuser/Mellon
```
Comment (mmoayyed) [inline etc/cas/config/management.properties]: Please change this to apereo. There is no such thing as jasig anymore

**Comment by mmoayyed** [comment]
Thanks! You want to bump this to 5.1 too? Should be straight forward.

**Comment by hdeadman** [comment]
I was going to bump to 5.1 after you merged this so it could be on both branches (5.0/master) but I could include it in this if you prefer.

**Comment by mmoayyed** [comment]
Cool. Please do it after this then.

**Pull request merged**